ambry-store/src/test/java/com.github.ambry.store/CuratedLogIndexState.java

```diff
@@ -331,7 +376,8 @@ IndexValue getExpectedValue(MockId id, boolean wantPut) {
 
   /**
    * Gets an ID to delete from the index segment with start offset {@code indexSegmentStartOffset}. The returned ID will
-   * have been removed from {@link #liveKeys} and added to {@link #deletedKeys}.
+   * have been removed from {@link #liveKeys} and added to {@link #deletedKeys}. A call to
```

Comment:
Sorry, why isn't "Gets an ID to delete from the index segment" not right anymore?

I'm not sure if documentation needs to call out implementation details. Do you think there is a way the caller would benefit from that information?